code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -10,14 +10,37 @@
 	req_one_access = list(access_armory)
 	var/locked = 1
 	var/broken = 0
+	var/startswithelectronics = TRUE
 	var/icon_locked = "lockbox+l"
 	var/icon_closed = "lockbox"
 	var/icon_broken = "lockbox+b"
 	var/tracked_access = "It doesn't look like it's ever been used."
+	var/obj/item/weapon/circuitboard/airlock/electronics = null
 	health = 50
+	starting_materials = list(MAT_PLASMA = 1000, MAT_IRON = 1875)
+
+/obj/item/weapon/storage/lockbox/New()
+	. = ..()
+	if(startswithelectronics)
+		electronics = new(src)
+		if(req_access?.len)
```

Comment:
Why are you checking `len` here? What's the edge case if `req_access = list()`

Reply:
conf access starts off as null, so there doesn't need to even be one, len has been removed and just the base "if" is kept